### DISCUSSION TITLE
loosen fastapi pin?

### DISCUSSION THREAD

**Discussion by lostmygithubaccount** [reflex-dev/reflex#1949] [General]
currently FastAPI is pinned fairly aggressively, causing some issues with other packages I'm trying to use. could that be loosened to allow for pip install of the latest versions?

**Comment by picklelo** [comment]
What version of FastAPI are you trying to use? The latest version uses Pydantic v2.0 which we are still in the progress of upgrading to (see #1539) and then we should be able to support it.
